lib/cog/models/bundle_version.ex

```diff
@@ -55,7 +55,7 @@ defimpl Poison.Encoder, for: Cog.Models.BundleVersion do
 
     map = %{}
     |> Map.merge(Map.take(bundle, [:name]))
-    |> Map.merge(Map.take(bundle_version, [:version, :status]))
+    |> Map.merge(Map.take(bundle_version, [:version, :status, :description]))
 
     map = Map.update!(map, :version, &to_string/1)
```

Comment:
Along the lines of an earlier comment, do we not have any views that render this information?